lib/board.rb

```diff
@@ -4,80 +4,41 @@
 
```

Comment:
Always run and test your code before submitting for review..
i try to run your code but threw an weird error, see error below;
```
**Failure/Error: require 'paint'

LoadError:
  cannot load such file -- paint**
```
i guess you're trying to require `paint` for styling which obviously not working.